.circleci/config.yml

```
@@ -20,6 +20,21 @@ jobs:
           covermode: atomic
           failfast: true
           race: true
+  publish-bot:
+    environment:
+      NODE_AUTH_TOKEN: $NPM_TOKEN  # Use NPM_TOKEN as the auth token
+    docker:
+      - image: cimg/node:18  # Use Node.js 18
+    working_directory: ~/superchain-registry/
+    steps:
+      - checkout  # Checkout the code
+      - run:
+          name: Setup .npmrc for npm publishing
+          command: |
+            echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > ~/.npmrc
```

```suggestion
      - name: Set deployment token
        run: npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
        env:
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Using npm config set is more idiomatic way of doing this